ftests/core/src/main/java/org/commonjava/indy/ftest/core/lifecycle/HttpSiteConfigTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2011 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.ftest.core.lifecycle;
+
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.commonjava.indy.ftest.core.AbstractContentManagementTest;
+import org.commonjava.indy.ftest.core.AbstractIndyFunctionalTest;
+import org.commonjava.indy.model.core.StoreKey;
+import org.commonjava.indy.subsys.http.IndyHttpProvider;
+import org.commonjava.indy.subsys.http.util.IndySiteConfigLookup;
+import org.commonjava.indy.test.fixture.core.CoreServerFixture;
+import org.commonjava.util.jhttpc.model.SiteConfig;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Map;
+
+import static org.commonjava.indy.subsys.http.conf.IndyHttpConfig.DEFAULT_SITE;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class HttpSiteConfigTest
+        extends AbstractIndyFunctionalTest
+{
+    //TODO: How to get the injected siteConfigLookup? It is null...
```

Comment:
We would have to change AbstractIndyFunctionalTest to use a WeldBootInterface instead of just BootInterface. Then, we would have to expose / wrap WeldBootInterface.getContainer() and allow lookup of injected components. Not impossible, but not trivial either